docs/Style-Guide.md

```diff
@@ -23,132 +22,14 @@ _It is still on you to ensure your code conforms to the `standard` spec, whether
-@import "../../node_modules/bulma/sass/elements/container";
-@import "../../node_modules/bulma/sass/elements/content";
-@import 'form';
-@import "../../node_modules/bulma/sass/elements/icon";
-@import "../../node_modules/bulma/sass/elements/image";
-@import 'notification';
-@import "../../node_modules/bulma/sass/elements/other";
-// @import "../../node_modules/bulma/sass/elements/progress";
-@import "../../node_modules/bulma/sass/elements/table";
-@import "../../node_modules/bulma/sass/elements/tag";
-@import 'title';
-```
-
-#### How to override Bulma
-
-There are two ways of overriding Bulma:
-- **SCSS Variables** - provided by Bulma's framework.
-- **Classes** - Directly access Bulma's classes.
-
-When overriding a module, start by checking the module's documentation and see if there's a SCSS variable that does the job. Add the variables **before** the module import.
-
-```scss
-$message-radius: 1px;
-
-@import "../../node_modules/bulma/sass/components/message";
-```
-
-If there isn't a SCSS variable for that, the solution is to override directly the classes. Do that **after** the module import. When overriding a class, ask yourself if that should be modified everywhere by default or if it should be a new modifier.
-
-If it's a new modifier, add `.gi-` prefix. That brings at least 2 main benefits:
-- Easily understand if that code is Bulma's or ours.
-- Prevent Bulma's naming conflicts with our code on future updates.
-
-Let's see an example:
-
-```scss
-$message-radius: 1px;
-
-@import "../../node_modules/bulma/sass/components/message";
-
-.message-body {
-  // The default `border-width` value is `0 0 0 3px`.
-  // We want it to be `1px` everywhere, so we add a new modifier:
-  border-width: 1px;
-
-  // The default border-style is solid,
-  // We want to dashed it just on a particular view, so we add a new modifier:
-  &.gi-is-unfilled {
-    border-style: dashed
-  }
-}
-```
-
-```html
-<!-- We can easily distinguish our code from Bulma's -->
-<div class="message-body gi-is-unfilled"> ... </div>
-```
-
-#### Bulma's documentation
-
-Bulma sometimes changes *significantly*. Therefore it's important:
-
-- To access the correct documentation for the version of Bulma that we're using. (Check `package.json`, and then visit the version-specific docs.
-- **Never update Bulma "just because"!** Always create an issue to update Bulma first, and then create a single unique Pull Request only for that issue. You will have to verify and fix any UI-differences that might occur. Sometimes Bulma will change its API (remove/change classes, styles, variables, etc..). You must identify and fix all differences.
+In general, think simplicity and don't over ingenior.
+It's easier to add style in a component and add few lines of code than trying to create the most generic css that create increasing complexity.
+With design refinement the process should slowly reverse and we should see common pattern emerging naturally.
```

Comment:
I recommend deleting the 3 lines above, and moving the "TODO" comment on line 25 down here.